### ISSUE TITLE
Achilles concept count type mismatch

### ISSUE THREAD

**Issue by MaximMoinat** [OHDSI/WebAPI#2170]
## Observed behaviour
When running the query from endpoint `/ddl/achilles` on PostgreSQL, I get the error:
```
expected concept_id to have type int, but concepts.ancestor_id has type varchar
```

## Cause
For joining on the achilles results tables, the `ancestor_id` is indeed cast to varchar, but the recasting to int is not performed.
https://github.com/OHDSI/WebAPI/blob/3fd81ef2c21b6463f3bd52fea4856f47f1054bd2/src/main/resources/ddl/achilles/achilles_result_concept_count.sql#L22-L30

Which is used in the insert into here (`achilles_result_concept_count.concept_id` is of type int):
https://github.com/OHDSI/WebAPI/blob/3fd81ef2c21b6463f3bd52fea4856f47f1054bd2/src/main/resources/ddl/achilles/achilles_result_concept_count.sql#L106-L108

## Suggested solution
Replace line 108 with:
```sql
  CAST(concepts.ancestor_id AS INT)               concept_id,
```

**Comment by chrisknoll** [comment]
Yes that's correct. However, there's efforts to change the entire record count calculation so that we don't CTE the entire concept_ancestor table and store zeros for zero counts (the left joins at the end). But if we need a short term fix, what you propose is correct

**Comment by chrisknoll** [comment]
Should be fixed in #2172.
